waspc/examples/todoApp/headless-tests/tests/auth.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from "@playwright/test";
+import {
+  generateRandomCredentials,
+  performEmailVerification,
+  submitLoginForm,
+  submitSignupForm,
+} from "./helpers";
+
+test.describe("auth", () => {
+  // We need the login test to run after the signup test
+  test.describe.configure({ mode: "serial" });
+
+  const { email, password } = generateRandomCredentials();
+
+  test("social button renders", async ({ page }) => {
+    await page.goto("/signup");
```

Comment:
This test is named "social button renders". But where, in which context?
I see in the test that you go to `/signup` page. So you actually assume that as a context. Let's make that explicit somehow.

Also, I see below that other tests then assume they are already on the signup page. Which makes them dependent on this test (which kind of makes sense, it is running serial, but due to stuff above it is not very clear all together).

Maybe having one big describe that says we are on the signup page (and does the setup by moving there) would be enough? Or maybe each test wants to go to signup page to ensure that part? Not sure but we we should do something.